react-restaurant/src/Food.js

```diff
@@ -7,25 +7,28 @@ class Food extends React.Component {
     }
 
     componentDidMount() {
-        axios.get(`https://entree-f18.herokuapp.com/v1/menu/12`)
-            .then(response => {
-                console.log(response.data);
-                console.log(response.data.menu_items);
-                console.log(response.data.menu_items[0].description);
-                this.setState({ content: response.data.menu_items });
-                console.log(this.state.content)
-            })
+        if (localStorage.getItem(this.props.menuSection) === null) {
+            axios.get(`https://entree-f18.herokuapp.com/v1/menu/12`)
+                .then(response => {
+                    this.setState({ content: response.data.menu_items });
+                    
+                    // SetItem(key(each menu section), value)
```

Comment:
I feel like you could probably get rid of this comment, since I assume it was there to help structure the next line correctly